repos/fdbt-site/src/pages/searchOperators.tsx

```diff
@@ -142,31 +170,62 @@ export const renderSearchBox = (operatorsAdded: boolean, errors: ErrorInfo[]): R
     );
 };
 
-export const showSearchResults = (searchText: string, searchResults: Operator[], errors: ErrorInfo[]): ReactElement => {
+export const showSearchResults = (
+    searchText: string,
+    searchResults: Operator[],
+    errors: ErrorInfo[],
+    selectedOperators: Operator[],
+    setSelectedOperators: React.Dispatch<React.SetStateAction<Operator[]>>,
+): ReactElement => {
     const addOperatorsErrors: ErrorInfo[] = [];
     errors.forEach((err) => {
         if (err.id === addOperatorsErrorId) {
             addOperatorsErrors.push(err);
         }
     });
+    const [searchResultsCount, setSearchResultsCount] = useState(searchResults.length);
```

Comment:
what if searchResults is undefined / null? this will cause an error
will searchResults always be an empty array if no results are found?

Reply:
line 273:  checks the value before calling the function this const is in.